app/controllers/coaches_controller.rb

```diff
@@ -8,26 +8,26 @@ def new_registration_form
   def create
     @coach = Coach.new
                 
-    @coach.password = params.fetch(:password, nil)
-    @coach.password_confirmation = params.fetch(:password_confirmation, nil)
+    @coach.password = params.fetch(:password)
+    @coach.password_confirmation = params.fetch(:password_confirmation)
                 
-    @coach.email = params.fetch(:email, @coach.email)
+    @coach.email = params.fetch(:email)
                 
```

Comment:
I'm wondering whether we _need_ newlines here between all of these attribute assignments. Do you think it enhances readability? Do we do it in `draft:resource`? If we do it between these, then why not between `password` and `password_confirmation`?

Reply:
I agree these assignments don't need the newlines between them.